<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rechargePage</class>
 <widget class="QWidget" name="rechargePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>充值</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FFFFFF</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>HYWenHei</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>请选择充值金额</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="0" column="1" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="buy30">
         <property name="font">
          <font>
           <family>汉仪文黑-85W</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,255);border:2px dashed #D3D3D3;border-radius:10px;padding: -1;color:gray</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/30.png</normaloff>:/image/30.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buy198">
         <property name="font">
          <font>
           <family>汉仪文黑-85W</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,255);border:2px dashed #D3D3D3;border-radius:10px;padding: -1;color:gray</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/198.png</normaloff>:/image/198.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="1" alignment="Qt::AlignTop">
        <widget class="QLineEdit" name="otherMoney">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>HYWenHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px groove #D3D3D3;border-radius:20px;padding: -1;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buy328">
         <property name="font">
          <font>
           <family>汉仪文黑-85W</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,255);border:2px dashed #D3D3D3;border-radius:10px;padding: -1;color:gray</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/328.png</normaloff>:/image/328.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="buy6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>汉仪文黑-85W</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,255);border:2px dashed #D3D3D3;border-radius:10px;padding: -1;color:gray</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/6.png</normaloff>:/image/6.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="buy98">
         <property name="font">
          <font>
           <family>汉仪文黑-85W</family>
           <pointsize>16</pointsize>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,255);border:2px dashed #D3D3D3;border-radius:10px;padding: -1;color:gray</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/98.png</normaloff>:/image/98.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>HYWenHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>其他金额</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="buy648">
         <property name="font">
          <font>
           <family>汉仪文黑-85W</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,255);border:2px dashed #D3D3D3;border-radius:10px;padding: -1;color:gray</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="image.qrc">
           <normaloff>:/image/648.png</normaloff>:/image/648.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="1" alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="confirm">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>HYWenHei</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(28,135,255);border:none;border-radius:20px;padding: -1;color:white;</string>
         </property>
         <property name="text">
          <string>确认充值</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
